test/test_client.py

```diff
@@ -862,6 +862,25 @@ def test_upload_missing_upload_permission(self):
         with pytest.raises(mwclient.errors.InsufficientPermission):
             self.site.upload(filename='Test', file=BytesIO(b'test'))
 
+    def test_async_without_filekey(self):
+        self.configure()
+
+        with pytest.raises(TypeError):
+            self.site.upload(filename='Test', file=BytesIO(b'test'), asynchronous=True)
+
+    def test_async_args(self):
+        self.configure()
+
+        self.site.upload(filename='Test', filekey='abc', asynchronous=True)
+
+        args, kwargs = self.raw_call.call_args
+        logging.debug("ASYNC ARGS: ", args)
```

Comment:
This `logging.debug` call seems to be a leftover from development. I think we should remove it.


```suggestion
```